.github/actions/build-image/action.yaml

```diff
@@ -32,16 +32,12 @@ runs:
       shell: bash
       run: |
         touch dynatrace-operator-bin-sbom.cdx.json
-    - name: Set up QEMU
-      uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.6.0
     - name: Build target
       uses: redhat-actions/buildah-build@7a95fa7ee0f02d552a32753e7414641a04307056 # v2.13
       with:
         build-args: |
           GO_LINKER_ARGS=${{ steps.prep.outputs.go_linker_args }}
           GO_BUILD_TAGS=${{ steps.prep.outputs.go_build_tags }}
-          TARGETPLATFORM="linux/${{ inputs.platform }}"
-        context: .
```

Comment:
why did you remove it? yeah it's default value, but just curios.

Reply:
it is just noise, by default it is `.`